packages/components/src/components/accordion-group/accordion-group.stories.ts

```diff
@@ -13,6 +13,10 @@ const { generateTemplate } = storybookTemplate('sd-accordion-group');
  *  **Related templates**:
```

Comment:
sd-accordion-group

Used to group multiple accordions together.

Reply:
Updated